modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -97,21 +98,32 @@ type WalkerCallback = (node: AstNode) => void;
 const nonRcDataSpecialElements = new Set('style,xmp,iframe,noembed,noframes,script,noscript,plaintext'.split(','));
 const specialElements = new Set([ 'title', 'textarea', ...nonRcDataSpecialElements ]);
 
-const getPurifyConfig = (settings: DomParserSettings): Config => {
-  const config: Config = {
-    RETURN_DOM: true,
-    ALLOW_DATA_ATTR: true,
-    ALLOWED_URI_REGEXP: /^(?:(?:(?:f|ht)tps?|about|blob|file|mailto|tel|callto|cid|xmpp):|[^a-z]|[a-z+.\-]+(?:[^a-z+.\-:]|$))/i,
```

Comment:
This looks to have been added to try and support additional unknown protocols, however there's a separate setting for that so swapped to using it.